lib/compilers/swift.ts

```diff
@@ -62,4 +62,11 @@ export class SwiftCompiler extends BaseCompiler {
     override getIrOutputFilename(inputFilename: string): string {
         return this.getOutputFilename(path.dirname(inputFilename), this.outputFilebase).replace('.o', '.ll');
     }
+
+    override getDefaultFilters() {
+        return {
+            ...super.getDefaultFilters(),
+            commentOnly: false, // SIL output includes useful demangled commentary
```

Comment:
is there anything specific that might work differently (or not work) without comments being filtered out by default? i tested a bit on the live site and things looked alright to me, but i'm likely not aware of the possible places it might matter.

Reply:
This doesn't really do anything, this always gets overwritten to what's sent from the browser.

I think if this is only really for IR, the best thing you can do is override processIrOutput where you copy the filters and change them before calling the original processIrOutput.